nexus/db-model/src/vmm_state.rs

```diff
@@ -104,7 +106,7 @@ impl From<VmmState> for omicron_common::api::external::InstanceState {
             VmmState::Starting => Output::Starting,
             VmmState::Running => Output::Running,
             VmmState::Stopping => Output::Stopping,
-            VmmState::Stopped => Output::Stopped,
+            VmmState::Stopped | VmmState::SagaUnwound => Output::Stopped,
```

Comment:
nit: I might throw a comment in here too just to point out what this state means in the grand scheme of things

Reply:
how about this:
```suggestion
            // `SagaUnwound` should map to `Stopped` so that an `instance_view`
            // API call that produces an instance with an unwound VMM will appear to
            // be `Stopped`. This is because we instances with unwound VMMs can
            // be started by a subsequent instance-start saga, just like
            // instances whose internal state actually is `Stopped`.
            VmmState::Stopped | VmmState::SagaUnwound => Output::Stopped,
```

Reply:
Except for the extra "we" on line 116, lgtm ship it :)